### ISSUE TITLE
Bug in MultiplicativeNeutralElement for ideals

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#363]
```
gap> S := SingularFactorisableDualSymmetricInverseMonoid(3);
<inverse bipartition semigroup ideal of degree 3 with 1 generator>
gap> IsMonoidAsSemigroup(S);
true
```
This should return false. I'm fixing it.

**Comment by james-d-mitchell** [comment]
Yikes, good catch.

**Comment by wilfwilson** [comment]
Resolved by PR #364
